editors/code/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 
 ## Development version
 
+- The extension now activates automatically when an `air.toml`, `DESCRIPTION`, or `.Rproj` file is detected at the workspace root. Previously, the extension only activated after an R file was opened (#285).
```

Comment:
(Technically it also used to activate if any `*.R` file was detected at the workspace root, but I imagine that was rare for our current set of users and is not worth mentioning)